HashRegistrarSimplified.sol

```diff
@@ -104,7 +104,8 @@ contract Registrar {
     enum Mode { Open, Auction, Owned, Forbidden, Reveal }
     uint32 constant auctionLength = 5 days;
     uint32 constant revealPeriod = 48 hours;
-    uint32 constant initialAuctionPeriod = 4 weeks;
+    uint32 constant initialAuctionPeriod = 2 weeks;
```

Comment:
Do we need a longer initial period under this proposal? Removing it might simplify the whole thing.